ggml/src/CMakeLists.txt

```diff
@@ -489,9 +497,9 @@ if (GGML_SYCL)
 
     add_compile_options(-I./) #include DPCT
```

Comment:
You may want to remove this, since the headers are already included in llama and are living under ggml-sycl/helpers/